pkg/api-server/definitions/all.go

```diff
@@ -15,6 +14,5 @@ var All = []ResourceWsDefinition{
 	FaultInjectionWsDefinition,
 	CircuitBreakerWsDefinition,
 	ZoneWsDefinition,
-	ZoneInsightWsDefinition,
```

Comment:
Why it was deleted?

Reply:
because we already add all required endpoints for zoneInsight here https://github.com/kumahq/kuma/blob/master/pkg/api-server/server.go#L150

Reply:
yes, but that's overview. I may need to access only the insight, because I don't care about the mesh itself and it will be a lighter/quicker API operation. Also if we ever make a bug in overview (🤞 hope not) I can check individual items with API calls

Reply:
Oh okay, I see. But do you agree to make those `*InsighWstDefinitions` read-only?

Reply:
I'm on the fence about this. Do you remember the problem with blowing up ZoneInsight size? All you had to do is remove them and reconnect. With this change technically we are putting a constraint on ourselves.

Reply:
Yes, that makes sense, let's keep it.